transactions.c

```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+
+typedef struct BookTransaction
+{
+	int book_uid;
+	char* check_out_date;
```

Comment:
How we are going to accommodate splitting datetime field 
e.g struct Date
    int month
    int day
    int year